docs/src/modules/components/overview/tree-view/mainDemo/github/GitHubExample.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import Stack from '@mui/material/Stack';
+import GitHubTreeView from './GitHubTreeView';
+import GitHubFiles from './GitHubFiles';
+
+export default function GitHubExample() {
+  const [selectedItem, setSelectedItem] = React.useState<string | null>(null);
+
+  return (
+    <Stack sx={{ width: '100%', height: '100%', overflowY: 'auto' }} direction="row" spacing={0}>
+      <Stack
+        p={2}
+        sx={(theme) => ({
+          overflowY: 'auto',
+          borderRight: `1px solid ${theme.palette.divider}`,
+          height: '100%',
+          minWidth: 'fit-content',
+        })}
+      >
+        <GitHubTreeView selectedItem={selectedItem} setSelectedItem={setSelectedItem} />
+      </Stack>
+
+      <GitHubFiles selectedItem={selectedItem} setSelectedItem={setSelectedItem} />
```

Comment:
Wonder if we could just hide this part of the demo on small viewports so that the tree view part takes up the full width:

<img src="https://github.com/user-attachments/assets/4c549648-47f5-4d56-9863-69f2ab87023c" width="400" />

(same for the Figma one)